jenkins/catapult/runDockerTestsInnerTest.py

```diff
@@ -168,14 +169,18 @@ def main():
 		environment_manager.set_env_var('LD_LIBRARY_PATH', f'{output_path}/lib:{output_path}/deps')
 	logs_path = Path(args.out_dir) / 'logs'
 
+	# There seems to be a bug in gtest where specifying the `--gtest_output=` parameter causes
+	# llvm-symbolizer not to resolves its libraries.
+	# setting the test_output using environment variable works.
+	environment_manager.set_env_var('GTEST_OUTPUT', f'xml:{logs_path}{sep}')
```

Comment:
shouldn't this be `base_output_filepath` like previous?

Reply:
That's also an option but would need to set the environment variable for each test run.

If ``GTEST_OUTPUT`` is set to the output directory then it will automatically add the test filename.

I can move it back into the loop if you want.

Reply:
it's fine, just add comment as to behavior [it will automatically add the test filename]